appt-org/appt-website
# Replace "VoiceOver" in Voice Control guide

# What does this PR do?

Replaces "VoiceOver" with "Voice Control" in the Voice Control guide. I guess this is a typo or mistake to mention VoiceOver in this part of the guide.

## How do you test this PR?

Check if [section "Turn off voice control"](https://appt.org/en/docs/ios/features/voice-control#turn-off-voice-control) mentions VoiceOver or Voice Control.

<img width="748" alt="Screenshot 2024-08-26 at 09 18 08" src="https://github.com/user-attachments/assets/8ac97aef-45ad-4f6f-926c-974c1355d1da">
